style.css

```diff
@@ -11,8 +11,7 @@ body {
   align-items: center;
   justify-content: center;
   background: #e5e5e5;
-  overflow-y: auto;
-  overflow-x: hidden;
+  overflow: hidden;
```

Comment:
- [ ] It's still not possible to scroll up and down in your portfolio due to the `overflow: hidden` on the `body` of your HTML. I imagine that you added this attribute to prevent the user to scroll up and down while the menu is open, so it would be better to add it only when the user opens the menu and remove it when he/she closes the menu.